### ISSUE TITLE
Project Update 02/21

### ISSUE THREAD

**Issue by enb34** [enb34/alice-in-translation#9]
This week we were able to find a Japanese copy of _Alice_, one which surprisingly does not have any copyright restrictions. We also revised and finalized our schema for marking up the two texts, and have begun marking up the English version. We have also started making a wireframe HTML for our project. 

In the upcoming week, we will be continuing to markup the texts and develop the HTML. For the most part, Dana will be handling the HTML while I (Emily) will be focusing on the markup of the texts.

**Comment by richiebful** [comment]
Out of curiosity, do you guys have any documentation for your linguistic analysis, especially how it relates to the schema? It would be cool to follow along a little with that, and take a look at your methods.

**Comment by gmd29** [comment]
I know in your overview it says you're going to focus on songs, wordplay, and nonsense words. I'm curious if you have any predictions of what difference you might find between the texts or if its solely exploratory. Do you think for example, there might be consistent phonetic differences certain phonemes in nonsense words in English correspond to certain phonemes in Japanese etc.?

**Comment by brucknerp** [comment]
I agree with @richiebful, I'm wondering what  all these attributes-- iu" | "ns" | "uc" | "pow" --represent and how you are going to use them.

Also, I took a look at the first chapter or so for each language, and I'm surprised how different the alligator song in ch 2 is, since the Japanese relies on a ton of onomatopoeia that isn't really there in the English. I suspect that will be a recurring theme, as the Japanese language has much more onomatopoeia usage than English.

**Comment by Idi0teque** [comment]
The site looks awesome right now, if a bit sparse! What in particular are you going to be putting on it (in terms of tabs, content, whatever)? Also, with regards to Japanese speech, are there any preserved/translated puns in the Japanese translation? Would be interesting to see if you could mark those up.
